### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh [hunk 1]
```diff
@@ -62,7 +62,7 @@ for prj in "${no_prjs[@]}"; do
 	log_mustnot zfs set projectquota@$prj=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even that id is not existed"
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh]: Done!

[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh]: Ok, missed also here. Now it shoul be done.

File: tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh [hunk 2]
```diff
@@ -62,7 +62,7 @@ for prj in "${no_prjs[@]}"; do
 	log_mustnot zfs set projectquota@$prj=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even that id is nonexistent"
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh]: ok, should be done now.

I squash!

File: tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh (hunk 1 above)
[the remark this answers is not in the document]
Comment (freqlabs) [reply tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh]: Thanks!

File: tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh [hunk 3]
```diff
@@ -62,7 +62,7 @@ for prj in "${no_prjs[@]}"; do
 	log_mustnot zfs set projectquota@$prj=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even if that id is nonexistent"
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/projectquota/projectquota_004_neg.ksh]: This is fine. One other idea would be "can set all numeric id even if that id does not exist"

**Comment by Gelma** [comment]
> I think you changed the other instance to be "dose not exist", so let's do that here too.

Done! Thanks

**Pull request merged**